packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -200,6 +207,24 @@ function OrgSelect({
     }
   };
 
+  const fetchOrgsAsGlobalUser = (searchTerm: string) => {
```

Comment:
Is there a reason we use the terminology "GlobalUser" here, but "Admin" below?

Reply:
Habit, since we don't use the phrase "admin" in cloud code typically and instead refer to global users and global roles. I'll make it consistent for leafygreen's usages and keep everything to admin.